src/viewer.js

```diff
@@ -2127,7 +2141,7 @@ function getTileSourceImplementation( viewer, tileSource, imgOptions, successCal
         if ( tileSource.match( /^\s*<.*>\s*$/ ) ) {
             tileSource = $.parseXml( tileSource );
         //json should start with "{" or "[" and end with "}" or "]"
-        } else if ( tileSource.match(/^\s*[\{\[].*[\}\]]\s*$/ ) ) {
+        } else if ( _isJSON(tileSource) ) {
```

Comment:
I guess it's not a big deal, but it seems inefficient to parse the string twice. Maybe instead we should have a try/catch around a $.parseJSON (which we use instead of JSON.parse because we're still supporting IE8 (See https://github.com/openseadragon/openseadragon/issues/373)) here, and then test the result.